docs/_build/.gitignore

```diff
@@ -1 +0,0 @@
-*
```

Comment:
was this deliberate, btw?

Reply:
Oops, absolutely not. I played with force-rebuilding the docs a bunch of times and at some point the result of a `rm *` must have slipped through. I'll revert that.